packages/browserslist-config-edu/README.md

```diff
@@ -0,0 +1,29 @@
+# @chanzuckerberg/browserslist-config-edu
+
+Shared [Browserslist](https://browsersl.ist) config for CZI's Education initiative.
```

Comment:
What I'm mostly wondering is what the process looks like for updating a browser version for the entire initiative. Who do I go to verify that I can update a version before coming here to update the package?

Reply:
But yeah, maybe leaving it up to each app is the way to go?

Reply:
I'm not necessarily thinking they would diverge; I'm thinking that if I as a developer on a product team want to update the policy, how do I get buy-in so I can update the policy since it now applies to everyone? I think it would be useful to have some kind of guidance here to give the developer an idea of where to start.

Reply:
I think the guidance from https://go.czi.team/edu-browser-support would still apply

> Talk to the Frontend Infra team if there’s a feature that should be supported natively by browsers, and isn’t covered under the current browser support policy

Can't go wrong talking to the Frontend Infra team 😸. With or without this change, pre or post strategy shift, we'd certainly be involved somehow.